test/unit/DrawNoReplTest.cpp

```diff
@@ -0,0 +1,59 @@
+#include <catch.hpp>
```

Comment:
Would making one test case in one file with twice as many sections work as well? That would save creating the original datasets twice.